mlflow/cli.py

```diff
@@ -281,6 +282,34 @@ def server(backend_store_uri, default_artifact_root, host, port,
         sys.exit(1)
 
 
+@cli.command()
+@click.option("--backend-store-uri", metavar="PATH",
+              default=DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH,
+              help="URI to which to persist experiment and run data. Acceptable URIs are "
+                   "SQLAlchemy-compatible database connection strings "
+                   "(e.g. 'sqlite:///path/to/file.db') or local filesystem URIs "
+                   "(e.g. 'file:///absolute/path/to/directory'). By default, data will be deleted "
+                   "from the ./mlruns directory.")
+@click.option("--run-ids", default=None,
+              help="Comma separated list of runs to be permanently deleted.")
+def gc(backend_store_uri, run_ids):
+    """
+    Permanently delete runs in state deleted in the backend store.
+    """
+    backend_store = _get_tracking_store(backend_store_uri, None)
```

Comment:
I think we should use the `mlflow.tracking._get_store` method here instead (https://github.com/mlflow/mlflow/blob/9b73ef08c7bae1d49e1bb66d0c6aa8a5e2a0fe69/mlflow/tracking/_tracking_service/utils.py#L113).

Reply:
Done